TestApp/ios/Podfile.lock

```diff
@@ -1,4 +1,23 @@
 PODS:
+  - appcenter (2.1.0):
```

Comment:
I wonder why this file is in the repo.

Reply:
It's best practice to check in Podfile.lock to ensure consistency of the development environment (cocoapods version, dependency version, etc.). With RN 0.60 it's even more useful as RN.iOS natively uses cocoapods to manage native dependencies.